instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -350,11 +350,25 @@ def calculate_commenter_data(self) -> dict[str, Any]:
         }
 
         if self.database_system == "postgresql":
-            if hasattr(self.connect_module, "__libpq_version__"):
-                libpq_version = self.connect_module.__libpq_version__
-            else:
-                libpq_version = self.connect_module.pq.__build_version__
-            commenter_data.update({"libpq_version": libpq_version})
+            libpq_version = None
+            # psycopg
```

Comment:
Do we run any tests for pscopg 1? Like requirements-oldest we have in other packages.

Reply:
No we have only tests against mocks and that's why this got un-noticed. I should add something to the docker tests.

Reply:
Added the tests